.gitignore

```diff
@@ -1,32 +1,37 @@
-app/bootstrap.php.cache
-app/cache/*
-!app/cache/.gitkeep
-app/cmsdata/*
-!app/cmsdata/.gitkeep
-app/config/parameters.yml
-app/logs/*
-!app/logs/.gitkeep
-behat-parameters.yml
-bin/
-composer.phar
-vendor/
-web/assets/less/chameleon.css
-web/bundles/
-web/chameleon/mediapool/*
-!web/chameleon/mediapool/.gitkeep
+/var/cache/*
+!/var/cache/.gitkeep
+/var/cmsdata/*
+!/var/cmsdata/.gitkeep
+/config/parameters.yaml
+/var/log/*
+!/var/log/.gitkeep
+/behat-parameters.yml
+/bin/
+/composer.phar
+/public/assets/less/chameleon.css
+/public/chameleon/mediapool/*
+!/public/chameleon/mediapool/.gitkeep
 
-web/chameleon/outbox/*
-!web/chameleon/outbox/static/
-web/chameleon/outbox/static/*
+/public/chameleon/outbox/*
```

Comment:
This renaming of web to public -> doesn't this need more code changes as shown in this PR?

Or are all users of the "public directory" using already a (correctly set) constant instead?

Reply:
We use either PATH_WEB or $_SERVER['DOCUMENT_ROOT'], so there should be no problems (ironically we switched from public to web a few years ago, so we already know that we're "clean").

Nevertheless there will be references to the web directory in comments, docs and database-stored help texts which I didn't think of yet --> should be searched and replaced.